examples/pytorch/gat/README.md

```diff
@@ -1,49 +0,0 @@
-# Introduction
```

Comment:
Please, keep the readme files in examples and update instructions to reflect how to train models.